bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/commands/PipelineRunOptions.java

```diff
@@ -70,4 +70,8 @@ public class PipelineRunOptions extends PipelineOptions {
     @CommandLine.Option(names = { "-pc", "--pipeline-configuration" },
             description = "The Pipeline Configuration File when using the Jenkins Templating Engine")
     public File pipelineConfiguration;
+
+    @CommandLine.Option(names = { "-wlrc", "--write-log-retry-count" },
+            description = "Retry count for writing logs to output stream.")
```

Comment:
The description is not understandable.

```suggestion
    @CommandLine.Option(names = { "-wlrc", "--write-log-retry-count" },
            description = "Initializing build log forwarding to stdout may fail if the build log is not being created quickly enough. Therefore this option allows to configure the maximum number of retries in a retry loop with 100ms delay.")
```

Reply:
I improved description in [46a57b4](https://github.com/romanisb/jenkinsfile-runner/pull/2/commits/46a57b45ba37e7064e72d5fe0273106ca4e44233).